### PR TITLE
fixed WpfTextView leak

### PR DESCRIPTION
**Pull request by heejaechang**
we leaked a TextView and whole bunch of things that got put into its property bags and corresponding text buffer's property bags.

I am not 100% sure whether this is new or root cause of our VM regression but regardless this is a leak I found.

### CODE REVIEW

**Comment by heejaechang** [comment]
@rchande @pharring can you take a look? thank you

File: src/EditorFeatures/Core/Implementation/IntelliSense/ModelComputation.cs [hunk 1]
```diff
@@ -107,6 +107,9 @@ public virtual void Stop()
 
             // cancel all outstanding tasks.
             _stopTokenSource.Cancel();
+
+            // reset task so that it doesnt hold onto things like WpfTextView
```
Comment (pharring) [inline src/EditorFeatures/Core/Implementation/IntelliSense/ModelComputation.cs]: doesn't
(or "does not")

**Comment by pharring** [comment]
LGTM

**Comment by heejaechang** [comment]
@dotnet-bot test this please

**Comment by rchande** [comment]
:+1:

**Pull request merged**